packages/react-components/src/shared/src/useMergedRefs.ts

```diff
@@ -11,8 +11,8 @@ export function assignRef<T>(ref: ForwardedRef<T>, node: T) {
     }
 }
 
-export function mergeRefs<T>(...refs: ForwardedRef<T>[]) {
-    const mergedRef = (current: T) => {
+export function mergeRefs<T>(...refs: ForwardedRef<T>[]): (current: T) => void {
+    const mergedRef = (current: T): void => {
```

Comment:
Do you think we could somehow also exempt these local arrow functions from the rule?

Reply:
Not a big deal if we can't